[Term Entry] C++ Math Functions: sinh() #980 (#1047)
Codecademy/docs: content/cpp/concepts/math-functions/terms/sinh/sinh.md

---
Title: 'sinh()'
Description: 'Returns the hyperbolic sine of an argument given in radians.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Arithmetic'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

> Returns the hyperbolic sine of an argument given in radians.

Reviewer: ```suggestion
The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.
```

## Syntax

```cpp
std::sinh(n)
```

> ## Example 1

Reviewer: ```suggestion

## Example
```

> Use `sinh()` to return the hyperbolic cosine of `0.0` radians:

Reviewer: ```suggestion

Use `sinh()` to return the hyperbolic sine of `0.0` radians:

```

---
Title: 'sinh()'
Description: 'Returns the hyperbolic sine of an argument given in radians.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Arithmetic'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.

## Syntax

```cpp

> std::sinh(n)

Reviewer 2: ```suggestion
std::sinh(angle)
```

```

> 

Reviewer 2: ```suggestion

The `angle` parameter should be expressed in radians. To convert from degrees to radians:

```

And then I would add one more \```pseudo code snippet that details how to do this:

```pseudo
radians = degrees * PI / 180
```

## Example

> Use `sinh()` to return the hyperbolic sine of `0.0` radians:

Reviewer 2: ```suggestion
The following example uses `sinh()` to return the hyperbolic sine of `0.0` in radians:
```

---
Title: 'sinh()'
Description: 'Returns the hyperbolic sine of an argument given in radians.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Arithmetic'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.

## Syntax

```cpp
sinh(angle)
```

> The `angle` parameter should be expressed in radians. To convert from degrees to radians:

Reviewer 2: We seem to be missing the other side of the colon `:`?

Author: Done, Kindly check

---
Title: 'sinh()'
Description: 'Returns the hyperbolic sine of an argument given in radians.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Arithmetic'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.

## Syntax

> ```cpp

Reviewer 2: ```suggestion
```pseudo
```

sinh(angle)
```

The `angle` parameter should be expressed in radians. To convert from degrees to radians:

> ```

Reviewer 2: ```suggestion
```pseudo
```

radians = degrees * PI / 180
```

## Example

The following example uses `sinh()` to return the hyperbolic sine of `0.0` in radians:

```cpp
#include <iostream>

> #include <cmath>
> int main() {

Reviewer 2: ```suggestion
#include <cmath>

int main() {
```
